granulate_utils/metrics/yarn.py

```diff
@@ -16,10 +17,14 @@
 
 class ResourceManagerAPI:
     def __init__(self, rm_address: str):
+        self._info_url = f"{rm_address}/ws/v1/cluster/info"
```

Comment:
Why is it added here and also in YARN_CLUSTER_INFO_PATH? (same applies to other paths here I see..)